waku/waku.md

```diff
@@ -510,6 +523,15 @@ Known static nodes MAY also be used.
 
 ## Changelog
 
+### Version 0.4
+
+Released [February 20, 2020](https://github.com/vacp2p/specs/commit/dafe08ea8792fbb9afa0be67cda3c97c79a1de67).
+
+- Add Status Update code `0x02`
```

Comment:
Is this 0x02 or 0x22? Maybe I fat fingered it with suggestion, right now there is one too many lines anyway

Reply:
`0x2` I'll edit it